### PR TITLE
Added a file path to the unsecured route CLI example in the Creating Routes section to addess PR-779

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by mburke5678** [comment]
See: https://bugzilla.redhat.com/show_bug.cgi?id=1479919

**Comment by mburke5678** [comment]
@thoraxe PTAL

**Comment by mburke5678** [comment]
@thoraxe Can you take a look to see if this addresses your concerns in https://github.com/openshift/openshift-docs/issues/779 ?

**Comment by thoraxe** [comment]
#779 is about the lack of documentation. This PR includes a link to some documentation. So I guess the answer to your question is "sort of"?

**Comment by mburke5678** [comment]
@thoraxe Added  `path: "/test" <1>` to the Route object example. with _<1> For path-based-routes[path-based routing], specify a path component that can be compared against a URL._

Does this work?

**Comment by thoraxe** [comment]
LGTM!

**Comment by mburke5678** [comment]
zhaozhanqi 2017-08-29 01:49:44 EDT
The changes LGTM, Could you merge this PR, I will verify this bug after that.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL. Minor change to add data to the `path` field in a Route object example, per [Issue 779](https://github.com/openshift/openshift-docs/issues/779)

File: dev_guide/routes.adoc [hunk 1]
```diff
@@ -63,11 +63,15 @@ metadata:
...
 spec:
   host: www.example.com
+  path: "/test" <1>
   to:
     kind: Service
     name: frontend
 ----
 ====
+
+<1> For xref:..//architecture/core_concepts/routes.html#path-based-routes[path-based routing], specify a path component that can be compared against a URL.
```
Comment (bfallonf) [inline dev_guide/routes.adoc]: Double // before architecture

**Comment by bfallonf** [comment]
@mburke5678 One comment from me. LG otherwise!

**Comment by mburke5678** [comment]
[rev_history]
|xref:../dev_guide/routes.adoc#dev-guide-routes[Routes]
|Added a file path to the unsecured route CLI example in the xref:../dev_guide/routes.adoc#creating-routes[Creating Routes] section.
%

**Pull request merged**